### ISSUE TITLE
Gray bands due to notch in iphone 13 screen

### ISSUE THREAD

**Issue by boscomac** [monkeyswarm/MobMuPlat#83]
Due to the iPhone 13 screen notch, patches loaded in Mobmuplat show gray bands on the ends. Is there a way to change this color to black? (The objective is to avoid stray lighting)

**Comment by lokkiikkol** [comment]
I don't think it is only because of the screen notch, I think the aspect ratio is just not supported by MobMuPlat
